README.md

```diff
@@ -35,6 +39,14 @@ depot must be the path to a folder with `ca.pem` and `ca.key` files.
 
 If you don't already have a CA to use, you can create one using the `scep ca` subcommand.
 
+The server currently has two endpoints:
```

Comment:
This line is inaccurate. 

`/scep` is for micromdm's server (this repo)
`/certsrv/mscep/` (with slash) is the url for mirosoft's SCEP server.